config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/EntityBuilderHelper.java

```diff
@@ -32,23 +36,30 @@ static Entity getEntityWithNameMapping(String type, String name, String id, Elem
     }
 
     static Entity getEntityWithNameMapping(String type, String originalName, String name, String id, Element element, String guid) {
-        Entity entity = new Entity(type, originalName, name, id, element, guid);
+        Map<String, Object> properties = new HashMap<>();
+        properties.put(PROPERTY_BUNDLE_ENTITY_NAME, name);
+        properties.put(PROPERTY_GUID, guid);
+        Entity entity = new Entity(type, originalName, id, element, properties);
```

Comment:
Introduce **setProperty** method and use it to introduce properties into it.

Reply:
Done!